src/util/audiosignal.cpp

```diff
@@ -4,6 +4,31 @@
 
 namespace mixxx {
 
+// Definitions of static class constants are required by
+// LLVM CLang, but cause "multiply defined symbols" errors
+// in Visual Studio 2013. GCC is happy either with or
+// without those definitions.
+// TODO(XXX): Replace 'static const' with 'static constexpr'
+// in the corresponding .h file after upgrading to Visual
+// Studio 2015 and remove all definitions from this .cpp file.
+#if !defined(_MSC_VER) || (_MSC_VER >= 1900)
```

Comment:
"not Visual Studio OR Visual Studio 2015 and newer"

Is that what you intended? Did you mean "only Visual Studio earlier than 2015"? 

```
#if defined(_MSC_VER) && (_MSC_VER < 1900)
```